providers/shared/services/resource.ftl

```diff
@@ -0,0 +1,30 @@
+[#ftl]
+
+[#-- Is a resource part of a deployment unit --]
+[#function isPartOfDeploymentUnit resourceId deploymentUnit deploymentUnitSubset]
+  [#local resourceObject = getStackOutputObject(commandLineOptions.Deployment.Provider, resourceId)]
```

Comment:
I think this needs to be `commandLineOptions.Deployment.Provider.Name` 

https://github.com/codeontap/gen3/blob/8580ab5da9e7313d0e78762cad9ca2b7414eb356/engine/bootstrap.ftl#L8

Reply:
Oddly, this was working for me without the specificity, however I recognise that was undesirable.

I have corrected this in c9d516b